Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -85,6 +104,19 @@ public static IEnumerable<PluginPair> DotNetPlugins(List<PluginMetadata> source)
                 metadata.InitTime += milliseconds;
 
             }
+
+            if (erroredPlugins.Count > 0)
+            {
+                var errorPluginString = "";
+
+                erroredPlugins.ForEach(x => errorPluginString += x + Environment.NewLine);
+
+                Task.Run(() =>
+                {
+                    MessageBox.Show($"Uh oh, unable to load plugins:{Environment.NewLine}{errorPluginString}");
```

Comment:
I like it! 
Maybe we could also hint the user about taking a look into logs? Like:
`MessageBox.Show($"Uh oh, unable to load plugins:{Environment.NewLine}{errorPluginString}{Environment.NewLine}Please refer to the logs for more information.");`
Potentially you could also include some styling for the message box like: ```MessageBoxIcon.Warning```